src/views/DashboardView.vue

```diff
@@ -135,11 +135,12 @@ const toggleBoardsNav = () => {
   isNavOpen.value = !isNavOpen.value
 }
 
-const windowWidth = ref(window.innerWidth)
-const resizeObserwer = new ResizeObserver((entries) => {
+const { width } = useWindowSize()
+const windowWidth = ref(width)
+const resizeObserver = new ResizeObserver((entries) => {
   windowWidth.value = entries[0].contentRect.width
 })
-resizeObserwer.observe(document.body)
+resizeObserver.observe(document.body)
```

Comment:
You don't need it anymore as you have the width property

Reply:
Just use `width` instead of windowWidth

Reply:
The whole description on this thing says "Reactive window size" and I obviously didn't read it 🙈😅

**Corrected**